tests/subsys/logging/log_syst/src/main.c

```diff
@@ -129,13 +129,12 @@ void test_log_syst_float_data(void)
 
 #endif
 /* test case main entry */
-void test_main(void)
+ZTEST(log_syst, test_log_syst)
 {
-	ztest_test_suite(test_log_syst,
-		ztest_unit_test(test_log_syst_format_table_selection),
-		ztest_unit_test(test_log_syst_data),
-		ztest_unit_test(test_log_syst_float_data),
-		ztest_unit_test(test_log_syst_data_multiple_args)
-		);
-	ztest_run_test_suite(test_log_syst);
+	test_log_syst_format_table_selection();
+	test_log_syst_data();
+	test_log_syst_float_data();
+	test_log_syst_data_multiple_args();
```

Comment:
Please fixed it, do not put them together unless you understand it and you have to, thanks.

Reply:
> Please fixed it, do not put them together unless you understand it and you have to, thanks.

ok,I will.thanks